Verso/PolychromaticSite.lean

```diff
@@ -48,13 +49,41 @@ around 4 million cases.
 At the moment, we have achieved the first of these goals, and the computational part of the second
 is completed, but not the non-computational aspects of the proof.
 
+# Mathematical Background
+
+## The Polychromatic Number
+
+The central object of study is the *polychromatic number* $`p(S)` of a finite set $`S` of integers.
+This is defined as the maximum number of colours possible in an $`S`-polychromatic colouring. Since
+any $`S`-polychromatic colouring must hit every colour at every translate, the number of colours
+cannot exceed $`|S|`, so $`p(S) \leq |S|`.
+
+## Strauss' Conjecture
+
+The *Strauss function* $`m(k)` is defined as the smallest $`m` such that every set of size at least
+$`m` has a polychromatic $`k`-colouring. Equivalently, $`m(k) \leq m` if and only if every set of
+size at least $`m` has $`p(S) \geq k`.
+
+Strauss conjectured that $`m(k)` is well-defined for all $`k` -- that is, for any $`k`, there exists
+some threshold $`m` beyond which all sets admit $`k`-polychromatic colourings. This was proved by
+Erdős and Lovász using the local lemma in 1975.
+
+## Newman's Conjecture
+
+Newman conjectured a specific value: that $`m(3) = 4`, meaning every set of 4 integers admits a
+3-polychromatic colouring. The upper bound $`m(3) \leq 4` was established by Axenovich, Goldwasser,
+Lidický, Martin, Offner, Talbot, and Young through a combination of theoretical reductions and
+computational verification. The lower bound $`m(3) > 3` follows from the fact that the set
+$`\{0, 1, 3\}` has polychromatic number exactly 2.
+
 The following diagram demonstrates the dependencies of these results.
 We define $`p(S)` to be the largest number of colours possible for an $`S`-polychromatic colouring,
 and $`m(k)` to be the smallest $`m` such that every set of size at least $`m` has a polychromatic
 $`k`-colouring. In this language, $`m(k) \leq m` if and only if every set of size at least $`m` has
-$p(S) \geq k$.
-Note that any $`S`-polychromatic colouring must use at least $`\#S` colours, so $`p(S) \leq \#S`,
-and in particular is defined.
+$`p(S) \geq k`.
+Note that any $`S`-polychromatic colouring must use at most $`|S|` colours (since each translate
+contains exactly $`|S|` elements and must hit every colour), so $`p(S) \leq |S|`, and in particular
+is defined.
 However it is not as immediate that $`m(k)` is well-defined: Strauss' conjecture says it is.
```

Comment:
This paragraph is now somewhat duplicated earlier, so remove or condense this one.